README.md

```diff
@@ -1,2 +1,63 @@
 # mobilefirstportfolio
 This is a portfolio website that has prioritized mobile user  experience over Desktop user.
+## Built With
+
+- HTML and CSS.
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+-Coming soon
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Bor Nelson**
+
+- GitHub: [@githubhandle](https://github.com/borrnelsonk)
+- Twitter: [@twitterhandle](https://twitter.com/bornelsonn)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bornelson/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- The Microverse community.
+-My standup team.
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your project doesn't have an MIT file. Please remove this section or create an MIT license file for your application so this doesn't redirect the user to a page that doesn't exist.